content/docs/start/experiments.md

```diff
@@ -54,92 +128,84 @@ reflected in the experiment run.
-scores.json  avg_prec  0.56191  0.009322
-scores.json  roc_auc   0.93345  0.018087
-
-Path         Param                   Value    Change
-params.yaml  featurize.max_features  3000     1500
+Path          Metric                Value    Change
+metrics.json  PR                    0.96257  0.0024179
+metrics.json  ROC                   0.99344  4.2617e-05
+metrics.json  categorical_accuracy  0.9175   0.0074
+metrics.json  false_negatives       927      -58
+metrics.json  false_positives       710      -51
+metrics.json  loss                  0.23387  -0.010393
+metrics.json  precision             0.92743  0.0052689
+metrics.json  recall                0.9073   0.0058
+metrics.json  true_negatives        89290    51
+metrics.json  true_positives        9073     58
+
+Path         Param                 Value    Change
+params.yaml  model.cnn.conv_units  32       16
 ```
 
 ## Queueing experiments
 
-So far, we have been tuning the `featurize` stage, but there are also parameters
-for the `train` stage (which trains a
-[random forest classifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html)).
+Instead of running the experiments one-by-one, we can define them without
+executing. This is especially handy when you have long running experiments to
+try.
 
-These are the `train` parameters from `params.yaml`:
-
-```yaml
-train:
-  seed: 20170428
-  n_est: 50
-  min_split: 2
-```
-
-Let's setup experiments with different hyperparameters. We can use the `--queue`
-flag to define all the combinations we want to try without executing anything
-(yet):
+We add experiments to the queue using `--queue` option of `dvc exp run`. Here,
+we also set the names of experiments to observe the results clearly:
 
 ```dvc
-$ dvc exp run --queue -S train.min_split=8
-Queued experiment 'd3f6d1e' for future execution.
-$ dvc exp run --queue -S train.min_split=64
-Queued experiment 'f1810e0' for future execution.
-$ dvc exp run --queue -S train.min_split=2 -S train.n_est=100
-Queued experiment '7323ea2' for future execution.
-$ dvc exp run --queue -S train.min_split=8 -S train.n_est=100
-Queued experiment 'c605382' for future execution.
-$ dvc exp run --queue -S train.min_split=64 -S train.n_est=100
-Queued experiment '0cdee86' for future execution.
+$ dvc exp run --queue -n cnn-48 -S model.cnn.conv_units=48
+$ dvc exp run --queue -n cnn-64 -S model.cnn.conv_units=64
+$ dvc exp run --queue -n cnn-96 -S model.cnn.conv_units=96
+$ dvc exp run --queue -n cnn-128 -S model.cnn.conv_units=128
 ```
 
 Next, run all (`--run-all`) queued experiments in parallel (using `--jobs`):
 
 ```dvc
 $ dvc exp run --run-all --jobs 2
+...
+Reproduced experiment(s): cnn-128, cnn-96, cnn-64, cnn-48
+To apply the results of an experiment to your workspace run:
+
+	dvc exp apply <exp>
+
+To promote an experiment to a Git branch run:
+
+	dvc exp branch <exp>
 ```
 
+The command will run all queued experiments in a random order.
```

Comment:
This again seems unnecessary since we already state above that this command will run all experiments. Is the ordering so confusing that it needs to be mentioned here? I would think many people wouldn't notice or care, and people familiar with parallel processing would expect random ordering. It's not much to keep if you think it will avoid confusion, but it's all about trying to keep this doc concise.